tests/envs/test_envs.py

```diff
@@ -55,6 +55,64 @@ def test_env(spec):
     env.close()
 
 
+def test_reset_info():
+    # blackjack gets its own assert statements because it returns a tuple as an obs, creating ambiguity for the normal assert statements
+    env_names = [
```

Comment:
I'm pretty sure there's a pattern somewhere in tests to automatically run tests on all valid envs, I think it's better to use that. This would need to be updated whenever one of these envs needs a version bump.

Reply:
Good catch, I made this change.